_sub/compute/k8s-fluxcd/vars.tf

```diff
@@ -23,6 +23,18 @@ variable "github_owner" {
   description = ""
 }
 
+variable "gitops_apps_repo_url" {
+  type        = string
+  default     = ""
```

Comment:
Would it not be better if these didn't have a default value? This default does not seem valid.

Reply:
These two vars have a default, but that is in the main module. The logic was that the gitops_apps_repo_url and gitops_apps_repo_branch could potentially be used by other main modules or by non-DFDS users.